code/turf/turf.dm

```diff
@@ -124,6 +124,30 @@
 
 	proc/selftilenotify()
 
+	proc/is_frozen()
+		var loc_temp = T0C
+		if(isturf(src.loc))
+			var/turf/T = src.loc
```

Comment:
same, no need to assign this, T here should just be src. 
temperature etc are local vars.